src/components/FeedModal/types.tsx

```diff
@@ -0,0 +1,5 @@
+export interface CarouselItemProps {
+  id: number;
+  level: number;
+  setActive: Function;
```

Comment:
It's acceptable, but would be great if we can be more specific with defining function types (how many arguments does it take, what types the function consumes and what does it return).

`foo: (arg1: number) => void`